hellogsm-web/src/main/java/team/themoment/hellogsm/web/domain/application/dto/request/ApplicationReqDto.java

```diff
@@ -92,12 +93,18 @@ public record ApplicationReqDto(
         String schoolName,
 
         @NotSpace
-        String schoolLocation
+        String schoolLocation,
+
+        @Pattern(regexp = "^(GENERAL|SOCIAL|SPECIAL)$")
+        @NotBlank
+        String screening
 ) {
     public Application toEntity(Long id, Long userId) {
         GraduationStatus graduationStatus = null;
+        Screening screening = null;
         try {
             graduationStatus = GraduationStatus.valueOf(this.graduation);
+            screening = Screening.valueOf(this.screening);
         } catch (IllegalArgumentException e) {
             throw new ExpectedException("graduation 값이 올바르지 않습니다", HttpStatus.BAD_REQUEST);
```

Comment:
에러 메세지가 적절하지 않습니다.
아래처럼 어느 필드가 문제인지 확실히 알려주도록 수정해야 합니다.
```suggestion
try {
    graduationStatus = GraduationStatus.valueOf(this.graduation);
} catch (IllegalArgumentException e) {
    throw new ExpectedException("graduation 값이 올바르지 않습니다", HttpStatus.BAD_REQUEST);
}
try {
    screening = Screening.valueOf(this.screening);
} catch (IllegalArgumentException e) {
    throw new ExpectedException("screening 값이 올바르지 않습니다", HttpStatus.BAD_REQUEST);
}
```

Reply:
깜빡했네요
수정 완료했습니다~